src/Rules/Swedish/Inflectible.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Inflector\Rules\Swedish;
+
+use Doctrine\Inflector\Rules\Pattern;
+use Doctrine\Inflector\Rules\Substitution;
+use Doctrine\Inflector\Rules\Transformation;
+use Doctrine\Inflector\Rules\Word;
+
+class Inflectible
+{
+    /** @return Transformation[] */
+    public static function getSingular(): iterable
+    {
+        yield new Transformation(new Pattern('/elar$/i'), 'el');
+        yield new Transformation(new Pattern('/kar$/i'), 'ke');
+        yield new Transformation(new Pattern('/kor$/i'), 'ka');
+        yield new Transformation(new Pattern('/mar$/i'), 'me');
+        yield new Transformation(new Pattern('/nar$/i'), 'n');
+        yield new Transformation(new Pattern('/nor$/i'), 'na');
+        yield new Transformation(new Pattern('/rar$/i'), 'er');
+        yield new Transformation(new Pattern('/sor$/i'), 'sa');
+        yield new Transformation(new Pattern('/tor$/i'), 'ta');
+        yield new Transformation(new Pattern('/der$/i'), 'd');
+        yield new Transformation(new Pattern('/or$/i'), 'a');
+        yield new Transformation(new Pattern('/ar$/i'), '');
+        yield new Transformation(new Pattern('/er$/i'), '');
```

Comment:
Which sort method did you use?

Reply:
In this case, we need to maintain a specific order. For example, "fingrar" must match `/rar$/` → "finger" before it can match the more general `/ar$/`, which would incorrectly yield "fingr." Similarly, "timmar" needs to match `/mar$/` → "timme" before `/ar$/` would produce "timm." This also applies to the plural list.